### PR TITLE
#16925857 create client README file

### PR DESCRIPTION
**Pull request by victkarangwa**
### What does this PR do?
This PR creates client README file for the app
#### Description of Task to be completed?
This PR Adds:
- [X]  Installation and run commands
- [X] Dependencies
- [X] Description
- [X] Tests setup and execution
- [X] Add badges
    Code Climate maintainability badge
    Code Climate code coverage badge
    CircleCI status badge
#### How should this be manually tested?
Simply navigate to branch's page:
[client README file](https://github.com/andela/team-odd-bn-frontend/tree/ch-create-client-README-169258573)
#### Any background context you want to provide?
Previously, the client didn't have a clue of this project. Nevertheless, the client has now a clear picture of how to run and experience the app
#### What are the relevant pivotal tracker stories?
[#169258573](https://www.pivotaltracker.com/story/show/169258573)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

**Review by nziokaivy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,38 @@
-[![Coverage Status](https://coveralls.io/repos/github/andela/team-odd-bn-frontend/badge.svg?branch=develop)](https://coveralls.io/github/andela/team-odd-bn-frontend?branch=develop)
-[![Maintainability](https://api.codeclimate.com/v1/badges/6a767cde4f78e69ece3c/maintainability)](https://codeclimate.com/github/andela/team-odd-bn-frontend/maintainability)
-[![CircleCI](https://circleci.com/gh/andela/team-odd-bn-frontend/tree/ch-update-circleCI-169258569.svg?style=svg)](https://circleci.com/gh/andela/team-odd-bn-frontend/tree/ch-update-circleCI-169258569)
+[![Coverage Status](https://coveralls.io/repos/github/andela/team-odd-bn-frontend/badge.svg?branch=develop)](https://coveralls.io/github/andela/team-odd-bn-frontend?branch=develop)   [![Maintainability](https://api.codeclimate.com/v1/badges/6a767cde4f78e69ece3c/maintainability)](https://codeclimate.com/github/andela/team-odd-bn-frontend/maintainability)   [![CircleCI](https://circleci.com/gh/andela/team-odd-bn-frontend/tree/ch-update-circleCI-169258569.svg?style=svg)](https://circleci.com/gh/andela/team-odd-bn-frontend/tree/develop)
```
Comment (hezronkimutai) [inline README.md]: - CircleCi is saying no build what could be the reason?

- I think you are missing dependencies and  in the readme

**Review by hezronkimutai — CHANGES_REQUESTED** [review]
Hey @victor left a comment

File: README.md (hunk 1 above)
Comment (victkarangwa) [reply README.md]: I think CirclrCI has no build because it was configured with `Update-circleCI` branch instead of `develop branch`

**Review by hezronkimutai — APPROVED** [review]
Nice work

File: README.md [hunk 2]
```diff
@@ -3,4 +3,47 @@
...
+1. Create a file and name it `.env` in root directory
+2. Find a file named `.env.example`
+3. Copy & paste all environmental variables from `.env.example` to `.env`
+4. Provide values to all environmental variables in `.env` file
+
+### Run commands
+Open terminal from your computer
+1. Run `npm install` to install all dependencies
+2. Run `npm test` to test automatically if the is functioning properly as expected
+3. To start the app, run `npm start`. You will see the following details:
```
Comment (NiyoEric) [inline README.md]: To start development server run `npm run dev`.
To build app run `npm run build` and run `npm start`

**Review by NiyoEric — CHANGES_REQUESTED** [review]
@victorkarangwa4 Nice Readme! Just check my feedback.

**Review by NiyoEric — APPROVED** [review]

**Review by shemaeric — APPROVED** [review]
Good work @victorkarangwa4

**Pull request merged**